### ISSUE TITLE
itch app stuck on disconnected second monitor

### ISSUE THREAD

**Issue by bjorn** [itchio/itch#1051]
Itch version: Can't tell
Operating system: Windows 10

I just launched itch after moving my PC over to another room, where I decided not to connect my second monitor. But itch stubbornly wants to sit outside of my screen and I can find no way to interact with it.

In the "Display settings" there is only my single screen configured.

Expected behavior is that itch does not restore to no longer existing desktop area.

**Comment by fasterthanlime** [comment]
Ah that's a fun one, let me try to reproduce.

In the meantime, you can work around it by pressing Shift-F12, then entering:

```javascript
require("electron").remote.require("electron").BrowserWindow.getAllWindows()[0].setPosition(0, 0)
```

...in the Console prompt and hitting enter.

That should move the window back to the top-left corner of your screen.

**Comment by bjorn** [comment]
That was fast, thanks!

Looks like it was rather more work than I had expected.

**Comment by fasterthanlime** [comment]
> That was fast, thanks!

No problem :) It'll drop in the next release.

> Looks like it was rather more work than I had expected.

True! Although I did go the extra mile to try and prevent any future similar problems from cropping up. On top of that, it was a Windows-only issue - how silly of me to assume Windows's desktop manager would reposition windows by itself if displays were removed :) Linux (XFCE at least) and macOS don't seem to suffer from that.
